README.md

```diff
@@ -19,7 +19,7 @@ Kong runs natively on Kubernetes thanks to its official [Kubernetes Ingress Cont
 
 Let’s test drive Kong by adding authentication to an API in under 5 minutes.
 
-We suggest using the docker-compose distribution via the instructions below, but there is also a [docker installation](https://docs.konghq.com/install/docker/) procedure if you’d prefer to run the Kong API Gateway in DB-less mode. 
+We suggest using the docker-compose distribution via the instructions below, but there is also a [docker installation](https://docs.konghq.com/mesh/latest/installation/docker/) procedure if you’d prefer to run the Kong API Gateway in DB-less mode. 
```

Comment:
Should it instead be:
https://docs.konghq.com/gateway/latest/install/

Reply:
Yes, your comment is more certain, it should be, but I recently figured out, from your comment, there is probably more specific link 
https://docs.konghq.com/gateway/3.1.x/install/docker/#install-kong-gateway-in-db-less-mode 
but it has hard-linked version and the latest link does not point to the same https://docs.konghq.com/gateway/latest/install/docker/#install-kong-gateway-in-db-less-mode 
I think the problem is in redirection, especially in fragment identifier of url and need to be fixed somewhere else. But, where? I don't know.